docs/SNAPTEL.md

```diff
@@ -184,14 +192,14 @@ and then:
 8. unload the plugins
 
 ```
-$ $SNAP_PATH/bin/snaptel plugin load $SNAP_PATH/plugin/snap-plugin-collector-mock1
-$ $SNAP_PATH/bin/snaptel plugin load $SNAP_PATH/plugin/snap-plugin-processor-passthru
-$ $SNAP_PATH/bin/snaptel plugin load $SNAP_PATH/plugin/snap-plugin-publisher-mock-file
-$ $SNAP_PATH/bin/snaptel plugin list
-$ $SNAP_PATH/bin/snaptel task create -t mock-file.json
-$ $SNAP_PATH/bin/snaptel task create -w workflow.json -i 1s -d 10s
-$ $SNAP_PATH/bin/snaptel task list
-$ $SNAP_PATH/bin/snaptel plugin unload -t collector -n mock -v <version>
-$ $SNAP_PATH/bin/snaptel plugin unload -t processor -n passthru -v <version>
-$ $SNAP_PATH/bin/snaptel plugin unload -t publisher -n publisher -v <version>
+$ snaptel plugin load /opt/snap/plugin/snap-plugin-collector-mock1
+$ snaptel plugin load /opt/snap/plugin/snap-plugin-processor-passthru
+$ snaptel plugin load /opt/snap/plugin/snap-plugin-publisher-mock-file
+$ snaptel plugin list
+$ snaptel task create -t mock-file.json
+$ snaptel task create -w workflow.json -i 1s -d 10s
+$ snaptel task list
+$ snaptel plugin unload -t collector -n mock -v <version>
+$ snaptel plugin unload -t processor -n passthru -v <version>
+$ snaptel plugin unload -t publisher -n publisher -v <version>
```

Comment:
@nanliu, could you check twice plugin unload command. I checked this trying to unload some loaded plugin and this command with flags **is no longer valid** as I supposed (see below my reasoning step by step):

a) load plugin
```
$ snaptel plugin load build/linux/x86_64/snap-plugin-collector-perfevents
Plugin loaded
Name: perfevents
Version: 8
Type: collector
Signed: false
Loaded Time: Wed, 11 Jan 2017 11:35:57 CET
```
b) unload plugin (deprecated command with flags)
```
$ snaptel plugin unload -t collector -n perfevents -v 8
Incorrect Usage: flag provided but not defined: -t

NAME:
   snaptel plugin unload - unload <plugin_type> <plugin_name> <plugin_version>

USAGE:
   snaptel plugin unload [arguments...]
flag provided but not defined: -t
```
**c) unload plugin with proper command**
```
$ snaptel plugin unload collector perfevents 8

Plugin unloaded
Name: perfevents
Version: 8
Type: collector
```
Based on that, please remove flags from all unloading plugin commands.